tasks/reverse_proxy.yml

```diff
@@ -1,45 +1,59 @@
 ---
+- name: remove deprecated nginx
+  docker_container:
+    name: "hub_nginx"
+    state: absent
+
+- name: remove deprecated nginx-gen
+  docker_container:
+    name: "hub_nginx_gen"
+    state: absent
+
 - name: Create a volume to hold the certificates for TLS / SSL
   docker_volume:
     name: "{{ volume_names.tls_certificates }}"
 
-- name: Start Reverse Proxy (nginx)
+- name: Start Reverse Proxy (traefik)
   docker_container:
     name: "{{ container_names.reverse_proxy }}"
     image: "{{ image_versions.reverse_proxy }}"
     restart_policy: always
     published_ports:
       - "80:80"
       - "443:443"
-    volumes_from:
-      - "{{ container_names.main }}"
+    command: |
+      --api.insecure=false
+      --api.dashboard=false
+      --api.debug=false
+      --log.level=DEBUG
+      --providers.docker=true
+      --providers.docker.exposedbydefault=false
+      --providers.file.filename=/dynamic.yaml
+      --providers.docker.network=traefik_proxy
```

Comment:
I thought the network's name is `internal_routing` not `traefik_proxy`. Shouldn't be working like this.